carepoint/models/cph/pathology_code_type.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-TODAY LasLabs Inc.
+# License MIT (https://opensource.org/licenses/MIT).
+
+from carepoint import Carepoint
+from sqlalchemy import (Column,
+                        String,
+                        )
+
+
+class PathologyCodeType(Carepoint.BASE):
+    __tablename__ = 'fdrfmlitd_10'
```

Comment:
Dude the steps I took to find this. You don't want to know. But Imagonna tell you anyways:

```
USE cph;
SELECT c.name AS ColName, t.name AS TableName
FROM sys.columns c
    JOIN sys.tables t ON c.object_id = t.object_id
WHERE c.name LIKE '%icd%';
```